packages/debug/src/browser/debug-command.ts

```diff
@@ -15,11 +15,21 @@ import { MAIN_MENU_BAR, MenuPath } from "@theia/core/lib/common/menu";
 import { DebugService } from "../common/debug-model";
 import { DebugSessionManager } from "./debug-session";
 import { DebugConfigurationManager } from "./debug-configuration";
+import { DebugProtocol } from "vscode-debugprotocol/lib/debugProtocol";
+import { DebugThreadSelectionService, ThreadSelection } from "./debug-selection-service";
 
 export const DEBUG_SESSION_CONTEXT_MENU: MenuPath = ['debug-session-context-menu'];
+export const DEBUG_SESSION_THREAD_CONTEXT_MENU: MenuPath = ['debug-session-thread-context-menu'];
 
 export namespace DebugSessionContextMenu {
     export const STOP = [...DEBUG_SESSION_CONTEXT_MENU, '1_stop'];
+    export const RESUME_ALL_THREADS = [...DEBUG_SESSION_CONTEXT_MENU, '3_resume_all_threads'];
```

Comment:
These actions should be represented by buttons.

Reply:
Where should the buttons be displayed? Still here? Or just registered to the top debug menu? I figured the context menu should be a good place to have these because it covers the entire session.

Reply:
Top debug menu will be ok for now.